### PR TITLE
chore(package): update eslint-plugin-react to version 6.10.2

### PR DESCRIPTION
**Pull request by stipsan**
Closes #484

https://greenkeeper.io/

### CODE REVIEW

**Comment by stipsan** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://ad78767b-7f11-434b-8cc2-fc852d33ec01.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/stipsan/uikit-react/branch/greenkeeper/eslint-plugin-react-6.10.2/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: src/stories/Input.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ storiesOf('Input', module)
   For more advanced use cases please check out [\`redux-form-uikit\`](http://form.uikit-react.io).
   `, () => (
     <form className="uk-margin-bottom uk-form">
-      <Input placeholder="Text input" /> &nbsp;
-      <Input danger defaultValue="danger" /> &nbsp;
-      <Input success defaultValue="success" /> &nbsp;
+      <Input placeholder="Text input" />
```
Comment (sureshHARDIYA) [inline src/stories/Input.js]: `&nbsp` is shown lint error, not sure, why?

Comment (sureshHARDIYA) [inline src/stories/Input.js]:
```
  <Button primary>Primary</Button>
      &nbsp;
```
Adding to separate line does not show error

Comment (sureshHARDIYA) [inline src/stories/Input.js]: It should be refactored like, 
```
<div className="uk-margin-bottom">
      <Button primary>Primary</Button>
      &nbsp;
      <Button success>Success</Button>
      &nbsp;
      <Button danger>Danger</Button>
      &nbsp;
      <Button link>Link</Button>
      &nbsp;
      <Button type="submit">Submit</Button>
    </div>
```

Comment (sureshHARDIYA) [inline src/stories/Input.js]: @stipsan : want me to refactor like example above?

Comment (stipsan) [inline src/stories/Input.js]: yes, do it like those examples instead of deleting `&nbsp;`

**Pull request merged**